cscs-checks/apps/cpmd/cpmd_check.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, scale):
             self.valid_systems += ['dom:gpu']
         else:
             self.num_tasks = 16
-            self.time_limit = (0, 20, 0)
```

Comment:
Is this a realistic time limit? It looks like a workaround. Can you check please the execution time and then possibly remove this completely?